lib/ReactCrop.js

```diff
@@ -543,6 +559,13 @@ class ReactCrop extends Component {
 	arrayToPercent(arr, delimeter = ' ') {
 		return arr.map(number => number + '%').join(delimeter);
 	}
+	
+	preventDefault(e) {
```

Comment:
Is this for IE? In a modern browser e.preventDefault() without checks should be ok I think so just wondering